test/Microsoft.NET.Build.Tests/GivenThatWeWantToReferenceAnAssembly.cs

```diff
@@ -26,6 +26,13 @@ public class GivenThatWeWantToReferenceAnAssembly : SdkTest
             string referencerTarget,
             string dependencyTarget)
         {
+            if (UsingFullFrameworkMSBuild)
```

Comment:
> So then why not just leave these tests alone? Again, changing these tests seems irrelevant to your current work.

Other changes that I made (using the bundled version of .NET Core instead of explicitly setting RuntimeFrameworkVersion) broke these tests on full Framework.  I couldn't just leave them alone because these tests were loading the version of the RuntimeFrameworkVersion to use from DependencyVersions.props, which was the incorrect version after I updated to a new .NET CLI.  I could have updated the version in DependencyVersions.props to match the bundled RuntimeFrameworkVersion, but I don't think that's cleaner than just skipping the tests here.